src/repo/TopicRepository.java

```diff
@@ -0,0 +1,12 @@
+package repo;
+
+import domain.Topic;
+
+public class TopicRepository extends BaseRepository<Topic>
```

Comment:
Nu ai nevoie de clasa asta daca nu ai metode in plus.Foloseste clasa BaseRepository in servicii si ii dai ca argument path-ul la clasa topic, ceva de genu:

<constructor-arg value="domain.Topic">